main-actions/src/main/scala/sbt/Tests.scala

```diff
@@ -308,11 +308,10 @@ object Tests {
       frameworks: Map[TestFramework, Framework],
       testLoader: ClassLoader,
       runners: Map[TestFramework, Runner],
-      discovered: Vector[TestDefinition],
+      o: ProcessedOptions,
```

Comment:
Since `sbt.Tests` is public, we have to keep backward binary compatibility.
Could you add an overload for this?

Reply:
I've added overload for `sbt.Tests.apply`, but I guess I was too eager to add overload also for `sbt.ForkTests.apply`?